scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -154,24 +156,6 @@
   {% endif %}
 {% endmacro %}
 
-{% macro footer() %}
```

Comment:
But I really like the idea of having just one imported macro in all the variants page